create/templates/checkout_ui_extension/next-steps.txt

```diff
@@ -0,0 +1,11 @@
+
+🔭 > Your extension bundles are available at: 
```

Comment:
Do we still need this? Previously this was needed so that they can grab the script url but it's no longer necessary. The build process will also output the bundle names and sizes right?

Reply:
@t6d ^^

Reply:
Bundle name will be output but from that you can't automatically infer the URL. The questions is, do you need to?  You don't really need to look at the contents of the URL and having more than one URL Is possibly confusing. I hadn't questioned the usefulness when I looked into this with @dmhenry. I just asked him to keep parity. But this is a good time to simplify things. Let's get rid of the bundle URLs altogether then.